File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/clinical-new/shared/dto/formfielddata.dto.ts
import { Questionary } from "./questionary.dto";
import { SingleSelectMultipleSelect } from "./singleselect-multipleselect.dto";
import { TextBoxFreeTypeNumber } from "./textbox-freetype-number.dto";

export class FormFieldData_DTO {
    textBoxFreeTypeNumber: TextBoxFreeTypeNumber[];
    singleSelectMultipleSelect: SingleSelectMultipleSelect[];
    questionary: Questionary[];
}

File: Code/Websites/DanpheEMR/Services/NewClinical/DTOs/FormFieldData_DTO.cs
using DanpheEMR.Controllers.Clinical_New.DTO;
using System.Collections.Generic;

namespace DanpheEMR.Services.NewClinical.DTOs
{
	public class FormFieldData_DTO
	{
		public List<TextBoxFreeTypeNumber_DTO> textBoxFreeTypeNumber { get; set; }
		public List<SingleSelectMultipleSelect_DTO> singleSelectMultipleSelect { get; set; }
		public List<Questionary_DTO> questionary { get; set; }
	}
}
